manifests/repo.pp

```diff
@@ -23,7 +23,9 @@
 # * Phil Fenstermacher <mailto:phillip.fenstermacher@gmail.com>
 # * Ricahrd Pijnenburg <mailto:richard.pijnenburg@elasticsearch.com>
 #
-class logstashforwarder::repo {
+class logstashforwarder::repo(
```

Comment:
I think a space in `repo(` to make it `repo (` would be more style-guide-happy. Not a huge deal.